tools/FakeItEasy.Build/Program.cs

```diff
@@ -104,7 +104,7 @@ public static void Main(string[] args)
                 DependsOn("build", "outputDirectory", "pdbgit"),
                 () =>
                 {
-                    var version = Read(ToolPaths.GitVersion, "/showvariable NuGetVersionV2", ".");
+                    var version = Read(ToolPaths.GitVersion, "/showvariable NuGetVersionV2");

```

Comment:
When `SimpleExec` is passed a working directory, it prints an extra line like so:

`Working directory: .`

Both this argument and this line are redundant, since `"."` simple refers to the current directory.

The other changes to remove `./` are pure refactoring (no observable effect).